speed-dreams/source-2.2.3/src/modules/userinterface/legacymenu/mainscreens/ResearcherMenu.cpp

```diff
@@ -342,7 +397,9 @@ void* ResearcherMenuInit(void* p_nextMenu)
 
     void* param = GfuiMenuLoad("ResearcherMenu.xml");
     GfuiMenuCreateStaticControls(s_scrHandle, param);
-
+    
+    // Choose black box control
+    m_blackBox = GfuiMenuCreateButtonControl(s_scrHandle, param, PRM_BLACKBOX, s_scrHandle, SelectFile);
```

Comment:
Clarify the variable and add whitespace between this and the other unrelated control.
```suggestion
    m_blackBoxDialogControl = GfuiMenuCreateButtonControl(s_scrHandle, param, PRM_BLACKBOX, s_scrHandle, SelectFile);
    
```